### DISCUSSION TITLE
No mail sent

### DISCUSSION THREAD

**Discussion by liorth** [leihs/leihs#1762] [Q&A / Community Support] [answered]
Hello there. I newly deployed leihs today followed the guide at fresh new debian 11 server. Then I realised, that no email can be sent. I created users and want the users to login for testing. But they don't get the mail for setting up the password. 

I then tried the live demo of the leihs and the demo instance also doesn't send emails like reset the password, approved orders etc. 

What am I doing wrong?

**Comment by nimaai** [comment]
@liorth can you check if sending of emails is enabled at all under `/admin/settings/smtp/` ?

**Reply by liorth** [reply]
I checked it, it is enabled. Or do I need to configure this setting? The system is quite untouched because we are new and want to test if it works good for us.

**Reply by nimaai** [reply]
ok, then it must be a problem with the other smtp settings i guess

**Comment by nimaai** [comment]
you could check in the DB directly what the error message is of the last email which was supposed to be sent:
on the server: `psql leihs`

```sql
select trials, code, error, message from emails order by created_at desc limit 1;
```

and also via systemd: `journalctl -u leihs-mail.service` if there are any errors/warnings in the log

**Reply by liorth** [reply]
This is driving me crazy. If i left the setting untouched, i see error connection failed. OK I guess i should config something like smtp relay here? Then i tried gmail. I need admin account. Ok I don't have it. Then I tried outlook: SendAsDenied: not allowed to send as leihs-mail@leihs.mywebsite.com. OK then i have to set it back to localhost. But what do I put here as user name then?

**Reply by nimaai** [reply, marked as answer]
```
id                   | 0
enabled              | t
address              | localhost
authentication_type  | plain
default_from_address | noreply@zhdk.ch
domain               | ausleihe.zhdk.ch
enable_starttls_auto | f
openssl_verify_mode  | none
password             |
port                 | 25
sender_address       |
username             |
```

this is our working config at ZHdK using `postfix` on the leihs server itself. i hope it helps

**Reply by liorth** [reply]
OK. Now I finally understand. It is just some basic ignorance from me...I didn't install postfix at all. Anyway, now it works. 

Besides, where can I find some documentation to help me solve other potential problems. Normally if I don't know postfix for example, I can still find useful Infos like where to see the log and how to basically configure it. But when it comes to leihs. I didn't find things like that. The WiKi are not for that purpose, right?

**Reply by nimaai** [reply]
there are also some technical pages to be found on the wiki which might be of some help. other than that there is this submodule: https://github.com/leihs/leihs_documentation. there are also the README files of the specific submodules. 

normally leihs should work out of the box. mailing functionality and authentication+user/group-sync adapters are the only parts which are institution specific and should be configured as such. there are some docs for the authentication/sync-part. smtp/mailing is missing, because it is assumed that the server operators know what they are using and how to configure leihs to integrate successfully.
